python_practice/calculator/simple_calc.py

```diff
@@ -49,17 +45,11 @@ def divide(
     :return int: the quotient will be an int with a remainder
     """
 
-    quotient = x / y
-
     if y == 0:
-        print('undefined')
-
-    return quotient
-
-    remainder = x % y
-    print(quotient + " r" + remainder + " is the quotient")
+        TypeError('Cannot Divide by 0')
+    else:
 
-    return quotient
+    return (x / y, x % y)
```

Comment:
To make it prettier, separate each statement by line, like you do with parameters.